tests/src/Context/LogicalThreadContextTest.cs

```diff
@@ -17,7 +17,7 @@
 //
 #endregion
 
-#if NET_4_5 || NETSTANDARD1_3
+#if NET_4_5 || NETCOREAPP1_0 || NETCOREAPP2_0
```

Comment:
Yes, because the test project builds an executable artifact for a specific platform. The most convenient way to test the `netstandard1.3` library would be with a project targeting `netcoreapp1.0` or `netcoreapp1.1`.